### ISSUE TITLE
distributed training dataloader setting

### ISSUE THREAD

**Issue by wjlyclll** [NovelAI/novelai-aspect-ratio-bucketing#3]
Thanks for publishing this useful work! May I ask a question will you keep different gpus training data with same aspect ratio (BucketManagers in different process share same seed) in one iteration during distributed training？Hope for your reply ~~

**Comment by finetunej** [comment]
Different ranks my be operating on a differently shaped batch.

**Comment by wjlyclll** [comment]
> Different ranks my be operating on a differently shaped batch.

Thank you for your reply, which has resolved a confusion I've had for a long time
